great_expectations/render/renderer/email_renderer.py

```diff
@@ -80,3 +80,7 @@ def _get_report_element(self, docs_link):
         else:
             logger.warning("No docs link found. Skipping data docs link in the email message.")
         return report_element
+
+    @override
+    def __eq__(self, other: object) -> bool:
+        return isinstance(other, EmailRenderer)
```

Comment:
Should we override the base renderer and just assert that the types are the same?

Reply:
I figured it might be simplest to just implement this here in case we ended up wanting any other branching down the line. Seems unlikely that we would though. IDK, would you prefer that approach? If so I'll update